src/helpers/ht_device_info.service.ts

```diff
@@ -1,67 +1,47 @@
-import 'clientjs';
 import * as Fingerprint2 from 'fingerprintjs2';
 import { ErrorCode, ErrorMessage, HT_Error } from '../model/errors';
+import {DeviceInfo, DeviceInfoParam} from "../model/deviceInfo";
 
 
 class DeviceInfoService {
-    private clientjs;
     private fingerprintComponents = [];
 
+    private fingerprint = null;
```

Comment:
I don't think we need to set this value to null unless you need to do so explicitly for some reason, then do it in the constructor

Reply:
Not seeing the harm, but I'll add it to the list of https://www.pivotaltracker.com/story/show/171322089